install/linux/build-rpm.py

```diff
@@ -72,7 +73,7 @@ def build(qubes=False):
         container_tar_gz.rename(container_tar_gz_bak)
     try:
         subprocess.run(["poetry", "build", "-f", "sdist"], cwd=root, check=True)
-        os.rename(dist_path / sdist_name, build_dir / "SOURCES" / sdist_name)
+        shutil.copy2(dist_path / sdist_name, build_dir / "SOURCES" / sdist_name)
```

Comment:
We have switched here because `os.rename` does not work across filesystem boundaries. Unlike Linux's `mv` command (which copies the file and then unlinks it, in the case of different filesystems), `os.rename`  simply fails to move the file. So, for compatibility reasons, a copy-unlink approach is better.

I have add a comment in bdf0204dd33730fc294c82bcd934a35f940da6e2 to clarify this (hopefully).